app/models/protocol.rb

```diff
@@ -129,6 +129,10 @@ def is_study?
   def virgin_project?
     selected_for_epic == nil
   end
+  # returns true if selected_for_epic is nil in the database
+  def selected_for_epic_is_nil_in_db?
```

Comment:
If you are looking for changes that just happened or not you can use the rails helpers.  Look at these http://apidock.com/rails/ActiveRecord/Dirty